.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2024 The Ground Authors.
```

Comment:
I assume this workflow will be used for all CI jobs? (build, test, lint, etc?) If so, should we rename it `ci.yml` or similar?

Reply:
Done